src/checkpoint/mod.rs

```diff
@@ -76,15 +77,21 @@ pub struct CheckpointSubsystem {
     /// The subsystem uses a `ReloadableConfig` to ensure that, at all, times, the subnets under
     /// management are those in the latest version of the config.
     config: Arc<ReloadableConfig>,
-    wallet_store: Arc<RwLock<Wallet>>,
+    fvm_keystore: Arc<RwLock<Wallet>>,
```

Comment:
We should figure out a better naming and add consistency to the implementation of both keystores. I feel they should have a common interface and similar namings.